lib/howl/modes/default_mode.moon

```diff
@@ -14,13 +14,13 @@ class DefaultMode
 
   code_blocks: {}
 
-  indent: (editor) =>
+  indent: (editor, lines=editor.active_lines) =>
     indent_level = editor.buffer.config.indent
     dont_indent_styles = comment: true, string: true
     buffer = editor.buffer
     current_line = editor.current_line
 
-    editor\transform_active_lines (lines) ->
+    editor\transform_lines lines, ->
```

Comment:
@shalabhc `transform_active_lines` always transforms *all* selected lines...even if one of them requires using a different sub mode. I changed the editor to call the correct method on the correct line, but that requires passing in the set of lines it should operate on, like here.